utils/graphene.py

```diff
@@ -0,0 +1,55 @@
+"""
+Programatically generate a bunch of Filters fields that allow us to have
+graphene.List types in filtersets.
+
+From https://github.com/graphql-python/graphene-django/issues/190
+"""
+
+import django.forms
+import django_filters
+import graphene
+from graphene_django.forms.converter import convert_form_field
+
+
+def _generate_list_filter_class(inner_type):
```

Comment:
oh, really cool! we were having troubles with this in venepaikka and decided to use arguments instead of filters for this. might be useful for us, if we will have to eventually add a FilterSet there in that ConnectionField. 👍

Reply:
I was pretty close to use arguments as well, but I stumbled upon this after a lot of googling around. :sweat_smile: